examples/clients/elphmod/run.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+import elphmod
+import ipi._driver.driver
+
+driver = elphmod.md.Driver.load("driver.pickle")
+
+ipi._driver.driver.run_driver(unix=True, address="localhost", driver=driver)
```

Comment:
I would leave this as an **alternative** option  adding clearly that it will only work  when i-PI has been installed through pip

Reply:
For now, I have fully removed this option to keep it simple. If you think it is useful, I can bring back the removed script `run.py` and add an alternative phony target to the makefile that uses it.

Reply:
OK, it is probably better to leave this as an alternative option, because it shows that also `elphmod.md.Driver` on its own (without the wrapper in `drivers/py/pes/elphmod.py`) works like the drivers defined in `drivers/py/pes/`. It can be tried using the new phony target `run2`, which is not used by default.